app/(componentsDisplay)/datetime/page.tsx

```diff
@@ -8,14 +8,13 @@ import Column from 'morning-react-ui/components/layout/Column';
 import Columns from 'morning-react-ui/components/layout/Columns';
 import Container from 'morning-react-ui/components/layout/Container';
 import Navigation from 'morning-react-ui/components/layout/Navigation';
-import { Datez } from 'morning-react-ui/types';
 import { Size } from 'morning-react-ui/utils/Enum';
 
 export default function Page() {
-  const [timeValue, setTimeValue] = useState<Datez>(null);
+  const [timeValue, setTimeValue] = useState<Date | null | undefined>(null);
```

Comment:
peut être créer un type pour les Date | null | undefined ? genre OptionnalDate ? :D

Reply:
Je suis mitigé la dessus, je trouve ça un peu plus galère d'utilisation avec ce type.
Genre j'ai ce genre d'erreur par exemple quand j'essaye de mettre juste null:
Argument of type 'null' is not assignable to parameter of type 'OptionalDate | (() => OptionalDate)'.ts(2345)
je suis donc obligé de tout convertir dans ce type